ethereum/sidecar/server.go

```diff
@@ -264,53 +267,54 @@ func RunServer(
 		server.logger.Info("gRPC server routine stopped")
 	}()
 
-	server.logger.Info(
-		"Waiting for the initial AssetsLocked sync before launching " +
-			"AssetsUnlocked routine",
-	)
+	id, err := server.bridgeContract.ValidatorIDs(accountAddress)
+	if err != nil {
+		panic(fmt.Sprintf("failed to get bridge validator ID: %v", err))
+	}
 
-	// Wait until the initial synchronization of the AssetsLocked routine is
-	// complete.
-	select {
-	case <-server.assetsLockedReady:
+	if id != 0 {
 		server.logger.Info(
-			"Initial AssetsLocked sync completed; proceeding with " +
-				"AssetsUnlocked routines",
+			"Account is an attesting validator. Waiting for the initial " +
```

Comment:
Let's keep all logs lowercase and use `;` instead of `.`.

Reply:
Done in b9f834dd9372274d9bd5e7fe4faeb6e7748b6a18.